src/components/OSCALMarkupProse.js

```diff
@@ -8,6 +8,17 @@ import ReactMarkdown from "react-markdown";
  */
 
 export function OSCALMarkupMultiLine(props) {
+  if (props.paragraphComponent) {
+    return (
+      <ReactMarkdown
+        components={{
+          p: props.paragraphComponent,
```

Comment:
I am worried what this does if `props.paragraphComponent` is `undefined`. I am unsure of the functionality of rewriting components if given a `undefined` value. We may want to put some kind of check. 
```suggestion
          p: props.paragraphComponent ? props.paragraphComponent :  p ,
```


I also think this might cause the `StyledToolTip` to have issues.

Reply:
`p: props.paragraphComponent ?? 'p'`

Reply:
Iff we need a check in the first place

Reply:
Thanks! My most recent commits address this